src/main/resources/static/api.yml

```diff
@@ -3649,6 +3649,12 @@ paths:
           schema:
             type: string
             format: uuid
+        - name: apArea
```

Comment:
I think maybe this should be called 'apAreaId'?

Reply:
Good call 👍